<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>401</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>701</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>580</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="step1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>120</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Step 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="LCDScreen">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>261</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="step2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>120</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Step 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="step3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>120</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Step 3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="step4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>440</y>
       <width>150</width>
       <height>120</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>120</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Step 4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="step5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>410</y>
       <width>211</width>
       <height>161</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Step 5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="statInd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Stat Ind</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="shockInd">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Shock Ind</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
